edx_sga/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,231 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Were you supposed to create this file?

The descriptive comments are outdated.